diff --git a/backend/open_webui/main.py b/backend/open_webui/main.py
index 7086a3cc..91bf2ba5 100644
--- a/backend/open_webui/main.py
+++ b/backend/open_webui/main.py
@@ -1453,6 +1453,8 @@ async def generate_title(form_data: dict, user=Depends(get_verified_user)):
     # If the user has a custom task model, use that model
     task_model_id = get_task_model_id(model_id)
     print(task_model_id)
+    if "o1" in task_model_id:
+        task_model_id = "gpt-4o-mini"

     model = app.state.MODELS[task_model_id]
